db/migrate/20231028000539_add_not_null_constraint_for_name_attribute_of_in_game_items.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddNotNullConstraintForNameAttributeOfInGameItems < ActiveRecord::Migration[7.1]
+  def change
+    change_column_null :armors, :name, false
+    change_column_null :books, :title, false
+    change_column_null :clothing_items, :name, false
+    change_column_null :ingredients, :name, false
+    change_column_null :jewelry_items, :name, false
+    change_column_null :misc_items, :name, false
+    change_column_null :potions, :name, false
+    change_column_null :properties, :name, false
+    change_column_null :staves, :name, false
+    change_column_null :weapons, :name, false
+  end
```

Comment:
An astute reviewer will notice that there are more changes in the migration file than in the `schema.rb` file. However, when I removed extraneous lines from this file and ran the migration again, `NOT NULL` constraints began to be _removed_ from the schema, so I added them back here.